docs/guide/list-of-differences.md

```diff
@@ -20,20 +20,21 @@ See a full list of differences between HyperFormula, Microsoft Excel, and Google
+| Functionality                                      | Examples                                                                  | HyperFormula                                                                                                                                                                                                                                                                                                                                                 | Google Sheets                                                                                                                  | Microsoft Excel                                                                                                                |
+|----------------------------------------------------|---------------------------------------------------------------------------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|--------------------------------------------------------------------------------------------------------------------------------|--------------------------------------------------------------------------------------------------------------------------------|
+| Dependency collection                              | A1:=IF(FALSE(), A1, 0)<br><br>ISREF(A1)                                   | Dependencies are collected during the parsing phase, which finds cycles that wouldn't appear in the evaluation.<br><br>`CYCLE` error for both examples.                                                                                                                                                                                                      | Dependencies are collected during evaluation.<br><br>`0` for both examples.                                                    | Same as Google Sheets.                                                                                                         |
+| Named expressions                                  | SALARY:=$A$10 COST:=10*$B$5+100<br>PROFIT:=SALARY-COST<br>A1:=SALARY-COST | Only absolute addresses are allowed<br>(e.g. SALARY:= $A$10).<br><br>Named expressions can be global or scoped to one sheet only.<br><br>They can contain other named expressions.                                                                                                                                                                           | Named expressions are not available.<br><br>Named ranges can be used to create aliases for addresses and ranges.               | Named ranges and scoped named expressions are available.                                                                       |
```

Comment:
Why did you decide to drop `and named ranges` from this row's Functionality column?

Reply:
Because a named range is a special case of a named expression. But after rethinking it - in this context it makes sense to keep "and named ranges" phrase.